### ISSUE TITLE
Add ability to set the minimum height of the field

### ISSUE THREAD

**Issue by engram-design** [verbb/vizy#175]
The minimum height is fixed for Vizy fields, it'd be good to be able to configure that in the field settings.

**Comment by MattAppleton** [comment]
Very big props for this request, please have a look at this Verbb :)

You can see here, we are making an inline video component — here's what it looks like with Redactor...
<img width="883" alt="Zircon-Screenshot 2022-12-22 at 17 58 41" src="https://user-images.githubusercontent.com/6295710/209197550-087bfdb8-a433-422e-a686-7fd2e6b812e1.png">

And here with Vizy — unnecessarily huge!
<img width="972" alt="Zircon-Screenshot 2022-12-22 at 17 58 47" src="https://user-images.githubusercontent.com/6295710/209197562-38e3d0dc-b21d-4549-81ef-ff1fc255be85.png">

We'd stick with Redactor but there's an issue with any sort of html view edits, it trashes the Redactor html tags and puts them into plain text.

And advice / tweak ideas much appreciated!

**Comment by engram-design** [comment]
Added for the next release. To get this early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-4 as 2.0.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by MattAppleton** [comment]
Thanks @engram-design  — not sure Im doing this right, I have updated composer, cleared caches etc, but I'm still seeing large text fields, do I need to set a custom config file to reduce the number of lines visible? If so, not sure what the .json values might be...?

See here, Id like a single-line simple view for this kind of caption ideally... even better the Vizy fields could be set to 'collapsed' if we wanted :)

thanks...

**Comment by MattAppleton** [comment]
<img width="932" alt="Zircon-Screenshot 2023-01-04 at 09 59 04" src="https://user-images.githubusercontent.com/6295710/210530744-cad6cf08-21f8-4b86-8651-377bbfa8edcd.png">

**Comment by engram-design** [comment]
There is a new setting ("Initial Rows") for controlling the number of lines a Vizy field shows by default in the field settings. Its default hasn't changed, so you'll need to update the field settings.

**Comment by MattAppleton** [comment]
Thanks @engram-design — hmmm... I don't see that yet — and this is what I have in composer's requirements `"verbb/vizy": "dev-craft-4 as 2.0.10",`

I think you are saying we should see something here, right?

<img width="892" alt="Zircon-Screenshot 2023-01-04 at 11 03 03" src="https://user-images.githubusercontent.com/6295710/210541673-550c0345-b4d7-4366-90c0-516de0cf82ff.png">

**Comment by engram-design** [comment]
Should certainly be there, just below the "Block Configuration" settings.

<img width="919" alt="image" src="https://user-images.githubusercontent.com/1221575/210677283-48ef1015-bee1-48e1-9eaa-ec5f5f7bca12.png">

**Comment by engram-design** [comment]
@MattAppleton My apologies! I completely forgot to notice I hadn't pushed those changes yet (I was working offline over the break), so that should be there now.

**Comment by MattAppleton** [comment]
Ah! thought I was going nuts :).  Trying again now...

**Comment by MattAppleton** [comment]
Just confirming — thats working nicely so far :)

**Comment by engram-design** [comment]
Fixed in [2.0.11](https://github.com/verbb/vizy/releases/tag/2.0.11)
